afs/models.py

```diff
@@ -353,11 +412,24 @@ def _create(self, data, files=None, extra_paths=[], form="json"):
         )
 
         if not files:
-            response = utils._check_response(
-                requests.post(
-                    url, params=dict(auth_code=self.auth_code), json=data, verify=False
+            if form in "json":
```

Comment:
What not just use `is` or `==`? I think `in` would be better used to `list` related operation.

Reply:
String `in` is for contains anothor string.
Inner function usage `is` or `==` might be better.

Reply:
If you're using `in`, what would happen in following cases:
```python
form = 'son'
form = 'on'
```

In most case, use `is` or `==` can avoid some unexpected error.